pkg/security/util.go

```diff
@@ -3,8 +3,7 @@ package security
 const (
 	UIDRangeAnnotation = "openshift.io/sa.scc.uid-range"
 	// SupplementalGroupsAnnotation contains a comma delimited list of allocated supplemental groups
-	// for the namespace.  Groups are in the form of individual group ids or a range of group ids.
-	// Range format is supported in the form of a Block which supports {start}/{length} or {start}-{end}
+	// for the namespace.  Groups are in the form of a Block which supports {start}/{length} or {start}-{end}
 	SupplementalGroupsAnnotation = "openshift.io/sa.scc.supplemental-groups"
```

Comment:
It was originally meant to stand for service account but is not applicable since an SA isn't directly assigned these values, the ns is.  I will remove it from the newest annotation to not perpetuate wrongness for the sake of consistency unless you disagree with that approach and would rather keep it consistent.